alpha/apps/kaltura/lib/myUrlUtils.class.php

```diff
@@ -0,0 +1,8 @@
+<?php
+class myUrlUtils
```

Comment:
We don't create my... files anymore, it's an old convention.
Also, you can place this method in asset utils and call it replaceSpecialChars.